app/src/main/java/eu/kanade/presentation/manga/components/MangaBottomActionMenu.kt

```diff
@@ -308,3 +314,73 @@ fun LibraryBottomActionMenu(
         }
     }
 }
+
+@OptIn(ExperimentalStdlibApi::class)
+@Composable
+fun FailedUpdatesBottomActionMenu(
```

Comment:
shouldn't this be in `presentation/updates/components...`?

Reply:
Well yeah probably, just did a 1-1 migration and did the least possible changes to get it to build. I'll look into specifics soon. (Probably should turn this into a draft but 🤷‍♂️)